community_images/consul/bitnami/scripts/server_coverage_script.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -x
+set -e
+
+# Available Scripts
+ls /opt/bitnami/scripts
+
+# consul acl token list
+# consul acl boostrap
+consul members
+consul info
+
+# Registering a test service
+consul services register /consul.d/sample_service.json
+consul reload
+sleep 10
+
+# Query our service using HTTP Api
+curl http://localhost:8500/vi/catalog/service/web
+
+# Checking for the healthy instances
+curl 'http://localhost:8500/v1/health/service/web?passing'
+
+# Removing service
+consul services deregister /consul.d/sample_service.json
+
+# Consul kv
+consul kv put redis/config/connections 5
+consul kv get -detailed redis/config/connections
```

Comment:
add a verification on the get value

Reply:
Could you please guide me how?

Reply:
we can verify it's value to be 5 for correctness

Reply:
I've copied the output to the file and printed only the number of connections. (Value = 5)